packages/components/src/clipboard-button/README.md

```diff
@@ -0,0 +1,28 @@
+ClipboardButton
+===============
+
+## Usage
+
+```jsx
+class MyClipboardButton extends React.Component {
```

Comment:
This example can also be expressed as functional component wrapped with withState higher-order component (HOC). I believe it is code this way in the existing codebase.